staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -285,7 +285,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	APIServerTracing: {Default: true, PreRelease: featuregate.Beta},
 
-	ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Beta},
+	ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
`pull-kubernetes-e2e-kind-beta-features` will enable all beta feature that is default false.

I changed here to make apiserver not enable ValidatingAdmissionPolicy FG when running `pull-kubernetes-e2e-kind-beta-features`.